src/ui/components/tables/CCStressTable.vue

```diff
@@ -219,7 +218,7 @@ export default class CCStressTable extends Vue {
       case 2:
         return 2
       case 1:
-        return 3
```

Comment:
This is the change that fixes the listed issue. This has been brought in line with https://github.com/massif-press/compcon/blob/master/src/ui/components/tables/CCStructureTable.vue#L279